whitelist.json

```diff
@@ -4,6 +4,12 @@
     "nightbot",
     "moobot",
     "wizebot",
+    "streamjar",
+    "streamdeckerbot",
+    "vivbot",
+    "dinu",
```

Comment:
dinu is already in the list

Reply:
yeah ik but I removed it then readded it